src/www/ui_tests/api/Models/GroupTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2020 Siemens AG
+ Author: Valens Niyonsenga <valensniyonsenga2003@gmail.com>
```

Comment:
Please fix the copyright year and remove Siemens AG name. Use yours instead.

Reply:
Done, thank you so much!.